bus-mapping/src/circuit_input_builder.rs

```diff
@@ -919,6 +972,38 @@ impl<'a> CircuitInputStateRef<'a> {
         Ok(())
     }
 
+    /// Push a [`StackOp`] into the [`OperationContainer`] with the next
```

Comment:
```suggestion
    /// Push a write type [`StackOp`] into the [`OperationContainer`] with the next
```

Reply:
updated in https://github.com/appliedzkp/zkevm-circuits/pull/497/commits/206d5fd8df436421e37e2a80bfa8b2965c487361 thanks